pandas/tests/io/test_excel.py

```diff
@@ -2182,6 +2182,8 @@ def test_write_cells_merge_styled(self, merge_cells, ext, engine):
             assert xcell_b1.font == openpyxl_sty_merged
             assert xcell_a2.font == openpyxl_sty_merged
 
+    @pytest.mark.xfail(PY35 and not PY36, reason='only fails on Linux?',
```

Comment:
why is this changing?

Reply:
Because I wasn't able to make this test work on PY35 regardless of the versions I tried.